WISE-Community/WISE
# Fixed issue where run statistics pages were showing error pages.

How to test:
From admin index page, make sure that List Runs run Today, This Week, This Month, and Runs by Activity links all work. Before they were showing a 500 error page.

Fixes #1819.
